PinkSea.Lexicons/com/shinolabs/pinksea/beginLoginFlow.json

```diff
@@ -0,0 +1,52 @@
+{
+	"$schema": "https://internect.info/lexicon-schema.json",
+	"lexicon": 1,
+	"id": "com.shinolabs.pinksea.beginLoginFlow",
```

Comment:
to be clear, this isn't a public API is it? third-party clients would probably be proxying via their PDS and the PDS would supply an appropriate service token that PinkSea can verify

Reply:
wouldn't that also make me hide putOekaki? should third party just rely on pushing raw to the PDS, with only the queries being public?

Reply:
(fwiw, i don't fully understand PDS proxying)

Reply:
I thiiink so, there's no other way unless you want to negotiate some sort of token bearer scheme for third-party clients to authenticate with the AppView directly.